doc/library/d3viz/index.txt

```diff
@@ -130,6 +130,19 @@ may not be perfect. In this case, you can press the ``Release node``
 button in the top-left corner to automatically arrange nodes. To reset
 nodes to their default position, press the ``Reset nodes`` button.
 
+You can also display the interactive graph inline in
+IPython using ``IPython.display.IFrame``:
+
+.. code:: python
+
+    from IPython.display import IFrame
+    d3v.d3viz(predict, 'examples/mlp.html')
+    Iframe('examples/mlp.html', width=700, height=500)
```

Comment:
I believe it should be possible to implement that; the tricky part is to make sure related resources are available, but it should be doable. It will likely require a custom IPython extension which users will have to `%load_ext` though (similar to https://github.com/jovyan/pythreejs). I'm interested in working on that, but it probably doesn't belong to this ticket. Theano IPython extension can do more cool things than just providing js/css fils for d3viz.

I think embedding JS and CSS in a html string won't work because IPython sanitizes HTML displayed via display.HTML. Just did a quick prototype - d3viz no longer works after embedding all scripts and styles when displayed with display.HTML.